contrib/core/actions/send_mail/send_mail

```diff
@@ -8,9 +8,12 @@ TO=$1
 shift
 SUBJECT=$1
 shift
+SEND_EMPTY_BODY=$1
+shift
 BODY="$@"
 
-${MAIL} <<EOF
+if [[ -z "${BODY// }" && $SEND_EMPTY_BODY == 'True' ]] || [[ -n $BODY ]]; then
```

Comment:
Since st2 translates to a python boolean True/true in yaml will always turn into `True` - I am pretty sure pyyaml deals with this for us. Plus we do perform validation of values at execution time. From the REPL -

```
>>> str(True)
'True'
>>> str(False)
'False'
```

TLDR; should only end up with the literal `True` or `False`